modules/nw-dns-view.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * dns/dns-operator.adoc
+
+[id="nw-dns-view-{context}"]
+= View the default DNS
+
+Every new {product-title} installation has a `dns.operator` named default. It
+cannot be customized, replaced, or supplemented with additional `dnses`.
+
+.Procedure
+
+Use the `oc describe` command to view the default `dns`:
+----
+$ oc describe dns.operator/default
+Name:         default
+Namespace:
+Labels:       <none>
+Annotations:  <none>
+API Version:  operator.openshift.io/v1
+Kind:         DNS
+<SNIP>
+Status:
+  Cluster Domain:  cluster.local <1>
+  Cluster IP:      172.30.0.10 <2>
+<SNIP>
+----
+
+<1> The Cluster Domain field is the base DNS domain used to contruct fully qualified
+Pod and Service domain names.
+<2> The Cluster IP is the address pods query for name resolution. The IP is constructed
+by taking `.10` from the cluster Service CIDR. To find the Service CIDR of your cluster,
```

Comment:
Might be clearer to say that the IP address is defined as the 10th address in the service CIDR range.